<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSolverWidget</class>
 <widget class="QDialog" name="CSolverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolbox">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pgStarting">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>306</width>
         <height>210</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Starting values</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbInitialVals">
          <property name="text">
           <string>Initial values</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbStartAnalC">
          <property name="text">
           <string>Anal. concs.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbStartFV">
          <property name="text">
           <string>Fixed value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbUndo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbInitialGuess">
          <property name="text">
           <string>Initial Guess</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgDamping">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>306</width>
         <height>210</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Damping</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pb_pcf">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PCF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>tolerance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_pcf_tol">
            <property name="text">
             <string>1e-3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Damping factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dsb_dampfactor">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbDamping">
          <property name="text">
           <string>Damping</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pbDamp1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Damp once</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dampspecies"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbFixOL">
          <property name="text">
           <string>Fix outliers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgFitting">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>293</width>
         <height>503</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Fitting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbScaling">
          <property name="text">
           <string>Scaling</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbDamping">
          <property name="text">
           <string>Damping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb0OffD">
          <property name="text">
           <string>Zero off-diagonal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbMask">
          <property name="text">
           <string>Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbStepLimiter">
          <property name="text">
           <string>Step limiter</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbForcer">
          <property name="text">
           <string>Forcer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbStepwise">
          <property name="text">
           <string>Stepwise</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbLogFit">
          <property name="text">
           <string>Log fitting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Refine</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="rb_RefAll">
             <property name="text">
              <string>All</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbRefInter">
             <property name="text">
              <string>Interval</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>From</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbRefFrom">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbRefTo">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbIter1">
          <property name="text">
           <string>iter 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbIter10">
          <property name="text">
           <string>iter 10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbIter100">
          <property name="text">
           <string>iter 100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbIter1000">
          <property name="text">
           <string>iter 1000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgPlot">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>306</width>
         <height>210</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Components</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QRadioButton" name="rb_plotall">
             <property name="text">
              <string>All</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_plotindependent">
             <property name="text">
              <string>Independent components</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgTools">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>306</width>
         <height>210</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tools</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QPushButton" name="pbSmooth">
          <property name="text">
           <string>Smooth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbInterpolate">
          <property name="text">
           <string>Interpolate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_zeronans">
          <property name="text">
           <string>Zero NaNs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="text">
           <string>Save Results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbFortran">
          <property name="text">
           <string>Fortran engine</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="canvas" native="true"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSolverWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSolverWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
